File: WS/C/Q10.c
#include <stdio.h>

int main(){

    int marks;
    printf("Enter Marks : ");
    scanf("%d", &marks);

    if(marks >= 60){
        printf("First Division \n");
    }
    else if(marks >= 50 && marks < 60){
        printf("Second Division \n");
    }
    else if(marks >= 40 && marks <  50){
        printf("Third Division \n");
    }
    else if(marks < 40 && marks >= 0){
        printf("Fail \n");
    }
    else{
        printf("Invalid Input \n");
    }

    return 0;
}

File: WS/Python/Q10.py
marks = int(input("Enter Marks : "))

if(100>= marks >= 60):
    print("First Division")
elif(60 > marks >= 50):
    print("Second Division")
elif(50 > marks >= 40):
    print("Third Division")
elif(40 > marks >= 0):
    print("Fail")
else:
    print("Invalid Input")
    
       